types/sass-true/index.d.ts

```diff
@@ -2,7 +2,8 @@
 // Project: https://github.com/oddbird/true
 // Definitions by: robertmaier <https://github.com/robertmaier>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-import type { Options } from 'sass';
+import type { renderSync } from 'sass';
+export type Options = Parameters<typeof renderSync>[0];
```

Comment:
The latest version of sass has still exported Options.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/sass/index.d.ts#L22

Reply:
The issue is that the latest version of the exported `Options` in sass are a generic now. So the sass-true types would need to add that generic as well.

So to fix it for the later versions, we'd need to change the sass true types to this
```
export function runSass(sassOptions: Options<'sync'>, trueOptions: TrueOptions): void;
```

This would however break sass-true when used with older versions of sass, as the Options type doesn't have the generic there.

I've looked at the source code of `sass-true` and saw that they are [always using](https://github.com/oddbird/true/blob/main/lib/main.js#L53) `renderSync`. So by inferring the Options type from the `renderSync` function, the sass version in use doesn't matter, as long as it still has a `renderSync` function - which it needs anyways for sass-true to work. If sass ever deprecates that function, we would need a new sass-true version with new types anyways.

Reply:
On a sidenote: the `Options` type was not 100% correct. `renderSync` uses a type called `LegacyOptions<'sync'>`.
This one's exported as well and would therefore also be an option instead of inferring the type based on the function